pkg/framework/dataplane/loopback.go

```diff
@@ -88,11 +84,11 @@ func (ldp *loopbackDataPlane) capture() bool {
 		//Commented below section as we start capture on all ports regardless of its type.
```

Comment:
Packet capture in loopback is done as part of p4rt packet-ins. As a result of that, capture() and captureOnPorts are both not useful here. Since capture() has to be defined as part of the interface definition, can we change capture() to something as below with appropriate comments as to why we have empty capture() method?
func (*loopbackDataPlane) capture() bool { return true }

Reply:
done